src/schema/sources.ts

```diff
@@ -283,6 +289,11 @@ export const typeDefs = /* GraphQL */ `
     """
     sourceByFeed(feed: String!): Source @auth
 
+    """
+    Get user's favorite sources
+    """
+    favoriteSources(userId: ID!, limit: Int): [Source]
```

Comment:
Missing `@auth`?

Reply:
Should be intended not to have `@auth` for the tooltip part when viewing other users.

Reply:
But you could have auth and see other users still, its just so that someone who is not logged in the platform can scrape and find favorites from user ids. That feels like it should not be possible?

Reply:
Anonymous users should have access to it. Lee is right. Imagine someone landing on a post page hovering over a profile, they expected to see the devcard

Reply:
Got it 👍